docs/core/diagnostics/eventpipe.md

```diff
@@ -0,0 +1,91 @@
+
+EventPipe then writes the serialized events in the `.nettrace` file format, either as a stream via Diagnostic Ports or directly to a file. To learn more about the EventPipe serialization format, refer to the [EventPipe format documentation](https://github.com/microsoft/perfview/blob/master/src/TraceEvent/EventPipe/EventPipeFormat.md).
+
+## EventPipe vs. ETW/LTTng
+
+EventPipe is part of the .NET runtime (CoreCLR) and is designed to work the same way across all the platforms .NET Core supports. This allows tracing tools based on EventPipe, such as `dotnet-counters`, `dotnet-gcdump`, and `dotnet-trace`, to work seamlessly across platforms.
+
+However, because EventPipe is a runtime built-in component, its scope is limited to managed code and the runtime itself. EventPipe cannot be used for tracking some lower-level events, such as resolving native code stack or getting various kernel events. If you use C/C++ interop in your app or you want to trace the runtime itself (which is written in C++), or want deeper diagnostics into the behavior of the app that requires kernel events (that is, native-thread context-switching events) you should use ETW or [perf/LTTng](./trace-perfcollect-lttng.md).
+
+Another major difference between EventPipe and ETW/LTTng is admin/root privilege requirement. To trace an application using ETW or LTTng you need to be an admin/root. Using EventPipe, you can trace applications as long as the tracer (for example, `dotnet-trace`) is run as the same user as the user that launched the application.
+
+The following table is a summary of the differences between EventPipe and ETW/LTTng.
+
+|Feature|EventPipe|ETW|LTTng|
+|-------|---------|---|-----------|
+|Cross-platform|Yes|No (only on Windows)|No (only on supported Linux distros)|
+|Require admin/root privilege|No|Yes|Yes|
+|Can get OS/kernel events|No|Yes|Yes|
+|Can resolve native callstacks|No|Yes|Yes|
+
+## Use EventPipe to trace your .NET application
+
+You can use EventPipe to trace your .NET application in many ways:
+
+* Use one of the [diagnostics tools](#tools-using-eventpipe) that are built on top of EventPipe.
+
+* Use [Microsoft.Diagnostics.NETCore.Client](https://github.com/dotnet/diagnostics/blob/master/documentation/diagnostics-client-library-instructions.md) library to write your own tool to configure and start EventPipe sessions yourself.
+
+* Use [environment variables](#trace-using-environment-variables) to start EventPipe.
+
+After you've produced a `nettrace` file that contains your EventPipe events, you can view the file in [`PerfView`](https://github.com/Microsoft/perfview#perfview-overview) or Visual Studio. On non-Windows platforms, you can convert the `nettrace` file to a `speedscope` or `Chromium` trace format by using [`dotnet-trace convert`](./dotnet-trace.md#dotnet-trace-convert) command and view it with [`speedscope`](https://www.speedscope.app/) or Chrome DevTools.
```

Comment:
If possible, can we include a screencap of one of these views of a trace?  It doesn't necessarily need to be in the PR, but having graphics would be nice.

Reply:
Will do the screenshots later when I go back to doing those.